benchmark-service/benchmark/static/css/autocomplete.css

```diff
@@ -0,0 +1,189 @@
+#autoComplete_wrapper {
+    position:relative;
+    display:inline-block;
+}
+#autoComplete {
+    position:relative;
+    padding:0 2rem 0 3.5rem;
+    height:2.1rem;
+    width:6rem;
+    font-size:1.2rem;
+    outline:0;
+    border-radius:3px;
+    border:.05rem solid;
+    /* color:rgba(255,255,255,0); */
+    color:rgba(0,0,0,1);
+    background-repeat:no-repeat;
+    background-size:1.2rem;
+    background-origin:border-box;
+    background-position:center;
+    transition:all .4s ease;
+    -webkit-transition:all -webkit-transform .4s ease;
+    text-overflow:ellipsis;
+}
+#autoComplete::placeholder {
+    color:rgba(192,192,192,0);
+    transition:all .3s ease;
+}
+#autoComplete:hover::placeholder {
+    color:rgba(192,192,192,0.3);
+    transition:all .3s ease;
+    -webkit-transition:all -webkit-transform .3s ease;
+}
+#autoComplete:focus::placeholder {
+    padding:.1rem .6rem;
+    font-size:1rem;
+    color:rgba(192,192,192,0.3);
+    transition:all .3s ease;
+    -webkit-transition:all -webkit-transform .3s ease;
+}
+#autoComplete::selection {
+    color:transparent;
+}
+#autoComplete:focus::selection {
+    color:#fff;
+    background-color:rgba(192,192,192,1);
+}
+#autoComplete:hover {
+    padding:0 2rem 0 3.5rem;
```

Comment:
I find this padding a bit roomy! What do you think of just `padding-left: 0.5em;`?
<img width="632" alt="Screen Shot 2020-09-03 at 12 18 49 PM" src="https://user-images.githubusercontent.com/106194/92140841-a0d49e00-eddf-11ea-9363-c1fc4fe39c76.png">